doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -900,6 +900,23 @@ Parametrized inductive types
      because the conclusion of the type of constructors should be :g:`listw A`
      in both cases.
 
+     This behavior can be turned off with the option :opt:`Uniform Inductive Parameters`.
```

Comment:
Since you are defining the option at this point, you should rather write:

```
.. opt:: Uniform Inductive Parameters

   When this option is set (it is off by default)...

```